homeassistant/components/ads/__init__.py

```diff
@@ -21,21 +23,25 @@
 DATA_ADS = 'data_ads'
 
 # Supported Types
+ADSTYPE_BOOL = 'bool'
+ADSTYPE_BYTE = 'byte'
+ADSTYPE_DINT = 'dint'
 ADSTYPE_INT = 'int'
+ADSTYPE_UDINT = 'udint'
 ADSTYPE_UINT = 'uint'
-ADSTYPE_BYTE = 'byte'
-ADSTYPE_BOOL = 'bool'
 
-DOMAIN = 'ads'
-
-CONF_ADS_VAR = 'adsvar'
-CONF_ADS_VAR_BRIGHTNESS = 'adsvar_brightness'
-CONF_ADS_TYPE = 'adstype'
 CONF_ADS_FACTOR = 'factor'
+CONF_ADS_TYPE = 'adstype'
 CONF_ADS_VALUE = 'value'
+CONF_ADS_VAR = 'adsvar'
+CONF_ADS_VAR_BRIGHTNESS = 'adsvar_brightness'
+
+DOMAIN = 'ads'
 
 SERVICE_WRITE_DATA_BY_NAME = 'write_data_by_name'
 
+TYPES = [ADSTYPE_INT, ADSTYPE_UINT, ADSTYPE_BYTE, ADSTYPE_DINT, ADSTYPE_UDINT]
```

Comment:
This isn't used.

Reply:
Is this used anywhere?

Reply:
Can we remove this?

Reply:
I don't think I inserted this type. I removed it in my test environment, and de beckhoff ads seneors are working fine.